org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/ClassFile.java

```diff
@@ -5764,6 +5764,18 @@ public byte[] getBytes() {
 		}
 		return this.bytes;
 	}
+
+	/**
+	 * Sets the actual bytes of the class file.
```

Comment:
Should this be called `internalSetBytes(..)`?

Also, mention that the byte array is stored as-is, and subsequent modification will be reflected and vice versa.